crates/rolldown_common/src/types/symbol_ref_db.rs

```diff
@@ -47,10 +47,15 @@ impl SymbolRefDbForModule {
     Self {
       owner_idx,
       root_scope_id: top_level_scope_id,
-      classic_data: scoping
-        .symbol_names()
-        .map(|_name| SymbolRefDataClassic { link: None, chunk_id: None, namespace_alias: None })
-        .collect(),
+      classic_data: IndexVec::from_vec(vec![
+        SymbolRefDataClassic {
+          link: None,
+          chunk_id: None,
+          namespace_alias: None,
+        };
+        scoping.symbols_len()
+      ]),
```

Comment:
https://github.com/oxc-project/oxc/blob/2c53a728b0128290260e3ce676e25a2a0d94ff56/crates/oxc_semantic/src/scoping.rs#L237-L255
They should have the same length